src/views/Vacancies/components/VacancyListItem.tsx

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import BackupIcon from "@material-ui/icons/Backup";
+import Button from "@material-ui/core/Button";
+import EventOutlinedIcon from "@material-ui/icons/EventOutlined";
+import PersonOutlinedIcon from "@material-ui/icons/PersonOutlined";
+import _get from "lodash/get";
+import _values from "lodash/values";
+
+const useStyles = makeStyles(theme => ({
```

Comment:
we'll reuse these horizontal boxes in other parts of the app. how can we share styles?